local-cli/generator-macos/templates/macos/HelloWorld-macOS/ViewController.h

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
We should not introduce new FB copyright headers in user files. See https://github.com/facebook/react-native/pull/27725